tasks/nginx.yml

```diff
@@ -4,6 +4,8 @@
     name: httpd_can_network_connect
     state: yes
     persistent: yes
+  # https://github.com/ansible/ansible/issues/18692
```

Comment:
aufs, selinux, upstart, systemd... VMs and containers are so different when you try to make them the same...